tests/.gitattributes

```diff
@@ -0,0 +1 @@
+data/** filter=lfs diff=lfs merge=lfs -text
```

Comment:
what is the rationale for this separate file?

Reply:
also I think there is a .gitattributes in data

Reply:
I do it when I in tests folder, does it matter? all files under tests/data should be in git-lfs

Reply:
do you suggest we manually combine this with the one outside this folder?

Reply:
remove this one and put the line into the outside one